FESS-GUI/mainwindow.cpp

```diff
@@ -522,19 +518,157 @@ void MainWindow::on_lineEditPassword_textEdited(const QString &password)
         ui->pushButton_ApplySettings->setEnabled(true);
 }
 
-
-
 void MainWindow::on_actionLock_frame_rate_at_30FPS_triggered(bool checked)
 {
-    dataTimer->stop();
     if(checked)
-        refreshRate = 33;
+    {
+        graphRefreshRate = 30;
+        graphRefreshTimer->setInterval(refreshRateToMS(graphRefreshRate));
```

Comment:
No reason really, just a simple copy, paste and modify when I wrote it. Moved it outside the else to decrease duplication.